support/common/src/main/kotlin/com/studentcenter/weave/support/common/vo/Email.kt

```diff
@@ -0,0 +1,18 @@
+package com.studentcenter.weave.support.common.vo
+
+/**
+ * @param value 이메일 주소 문자열
+ * @throws IllegalArgumentException 이메일 주소가 올바른 형식이 아닌 경우
+ */
+@JvmInline
+value class Email(val value: String) {
+
+    companion object {
+        const val VALIDATION_REGEX = "[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\\.[a-zA-Z]{2,}"
```

Comment:
p3) `.`,`+` 문자는 뺼까요?? 트리키한 문자들이라
```suggestion
        const val VALIDATION_REGEX = "[a-zA-Z0-9_%-]+@[a-zA-Z0-9.-]+\\.[a-zA-Z]{2,}"
```

Reply:
지금 확인해보니 regex가 아예 잘못되었네요 이거는 수정하고 다시 올릴게요

Reply:
음 이거 빼버리면 regex가 동작안하는데 좋은 regex가 있을까요?? 레퍼 계속 찾아봤는데 적당한게 없네요

Reply:
스읍 고민이네요... + 문자열로 이메일 중복되는 케이스랑 `.` 제거되고 메일 가는 이슈가 있어서 요거 2개를 그냥 내부 필터 로직을 만들까요?